build/opt/pdo/etc/template/eservice.toml

```diff
@@ -68,3 +68,21 @@ DataPath = "${data}"
 # BaseName is the root of the name used to store data
 # about the enclave. A 'enc' extension will be added
 BaseName = "${identity}"
+
+# --------------------------------------------------
+# EnclavePolicy -- configuration of CDI policy for the enclave
+# --------------------------------------------------
+[EnclavePolicy]
+# The DefaulDeny flag indicates whether the CDI policy
+# only allows contract code built using trusted toolchains.
+# When true, a list of one or more trusted compiler keys is expected
+DefaultDeny = 0
+
```

Comment:
small nit... shouldn't this be a boolean?

Reply:
Fixed